src/main/java/org/jboss/xavier/integrations/jpa/service/AnalysisService.java

```diff
@@ -104,6 +111,7 @@ public Integer countByOwner(String owner)
 
     public Page<AnalysisModel> findByOwnerAndReportName(String owner, String filterText, int page, int size)
     {
+        assertTrue(size <= paginationLimitMax);
```

Comment:
Same as above. We can reuse the code using https://github.com/project-xavier/xavier-integration/blob/e4293da7eac386a3a82f5ad611a851aebce25daf/src/main/java/org/jboss/xavier/integrations/route/ToBeanRouter.java#L48

Reply:
Well in fact no.... it's the route to-pagebean who takes care of it.
I have already modified the code adding you latest changes about pagination, to this PR